pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
@smarterclayton, apologies if I'm being dense, but what is the pattern you are suggesting we use here? Log error and then return? Or are you suggesting that we fork/upstream commander's Run function to return an error and push error handling into the command framework?